basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/jobs/SiteMembershipsSyncJob.java

```diff
@@ -2,10 +2,13 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.imsglobal.basiclti.BasicLTIUtil;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 
+import org.sakaiproject.component.cover.ServerConfigurationService;
```

Comment:
Don't use cover classes - these are no longer best practice - instead directly use the ComponentManager (there are examples in other BasicLTI classes, such as LessonsFacade